src/containers/dashboard/CrisisReportDashboardContainer.js

```diff
@@ -0,0 +1,204 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+import { Map } from 'immutable';
+import { Spinner } from 'lattice-ui-kit';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import type { RequestSequence } from 'redux-reqseq';
+
+import { loadDashboardData } from './DashboardActionFactory';
+import { SUBMISSION_STATES } from './DashboardReducer';
+
+import ButtonToolbar from '../../components/buttons/ButtonToolbar';
+import DropdownButton from '../../components/buttons/DropdownButton';
+import IncidentCharts from '../../components/dashboard/IncidentCharts';
+import OutcomeCharts from '../../components/dashboard/OutcomeCharts';
+import OverviewCharts from '../../components/dashboard/OverviewCharts';
+import SummaryStats from '../../components/dashboard/SummaryStats';
+import { SUMMARY_STATS } from '../../shared/Consts';
+
+const OptionRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const Flex = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+`;
```

Comment:
Doesn't there exist a common component like this already?

Reply:
There might be. I just copied all of this into a new file. I'll check to see if there's an existing component to replace this with.